tutorials/shaders/shader_reference/canvas_item_shader.rst

```diff
@@ -238,18 +241,19 @@ Below is an example of a light shader that takes a CanvasItem's normal map into
 | Built-in                         | Description                                                                  |
 +==================================+==============================================================================+
 | in vec4 **FRAGCOORD**            | Coordinate of pixel center. In screen space. ``xy`` specifies                |
-|                                  | position in window. Origin is lower-left.                                    |
+|                                  | position in window. Lower-left of the screen is the origin, ``(0.0,0.0)``.   |
```

Comment:
again, is that window or viewport?

Reply:
The origin for `FRAGCOORD` in `light()` might be the upper left, too? This shader which outputs the `light()` `FRAGCOORD` seems to increase down and to the left, just like the `FRAGCOORD` in `fragment()`.

![Godot_v4 3-stable_win64_UpaFokh9d3](https://github.com/user-attachments/assets/57bb3989-c96f-45cb-b94a-e6e6a0d0fb6f)
![Godot_v4 3-stable_win64_btnYCzoCrS](https://github.com/user-attachments/assets/9b90ca78-7839-440b-bfa7-398e186b65cb)

The setup for testing a subviewport is more complicated for Node2D, still working on that.

Reply:
Conclusion: the `light()` `FRAGCOORD` is also the coordinate in the viewport, not the window. And it's origin is the upper-left, like the `fragment()` `FRAGCOORD`. 
![Godot_v4 3-stable_win64_kPGb6eQdcU](https://github.com/user-attachments/assets/ab578b1d-a3b0-4849-ac32-3b435f5d2507)

https://github.com/user-attachments/assets/8715d983-3174-4695-a1ad-6dda94ae0cd7

Reply:
Updated the description for both instances of FRAGCOORD to match observed behavior. This still might be technically a little incorrect, though?